opc/provider.go

```diff
@@ -49,6 +49,13 @@ func Provider() terraform.ResourceProvider {
 				DefaultFunc: schema.EnvDefaultFunc("OPC_INSECURE", false),
 				Description: "Skip TLS Verification for self-signed certificates. Should only be used if absolutely required.",
 			},
+
+			"storage": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("OPC_STORAGE", false),
```

Comment:
This env var should likely be more clear. `EnvDefaultFunc` also returns a string, not a boolean. Should we use a non-default function to allow any non-nil value to enable authentication? 
ie: 
```
DefaultFunc: func() (interface{}, error) {
  if v := os.Getenv(k); v != "" {
    return true, nil
  }
  return false, nil
}
```
Thoughts?